examples/all-clusters-app/all-clusters-common/include/tls-certificate-management-instance.h

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/clusters/tls-certificate-management-server/tls-certificate-management-server.h>
+#include <app/util/config.h>
```

Comment:
why is this include needed?

Asking because this pulls in application-specific generated files ... which are available in examples (because it is an app) but needing it is generally unusual.

Reply:
Good call, not sure why this was here; I think at the beginning I was using something from there but then moved that elsewhere? 🤷‍♂️